charts/cryostat/templates/NOTES.txt

```diff
@@ -32,32 +32,34 @@
   ```
 {{- end }}
 
-{{- if not (empty $portForwards) }}
+{{- if $portForwards }}
 
-{{ $listNum }}. Forward local ports to the application's pod:
+{{ $listNum }}. Forward local port to the application's pod:
   ```
-  export POD_NAME=$(kubectl get pods -n {{ .Release.Namespace }} -l "app.kubernetes.io/name={{ include "cryostat.name" . }},app.kubernetes.io/instance={{ .Release.Name }}"  --sort-by=.metadata.creationTimestamp -o jsonpath="{.items[-1:].metadata.name}")
   kubectl -n {{ .Release.Namespace }} wait --for=condition=available --timeout=60s deploy/{{ include "cryostat.fullname" . }}
-  kubectl -n {{ .Release.Namespace }} port-forward $POD_NAME {{ join " " $portForwards }}
+
+  export POD_NAME=$(kubectl get pods -n {{ .Release.Namespace }} -l "app.kubernetes.io/name={{ include "cryostat.name" . }},app.kubernetes.io/instance={{ .Release.Name }}" --sort-by=.metadata.creationTimestamp -o jsonpath="{.items[-1:].metadata.name}")
+  export CONTAINER_PORT=$(kubectl get pod -n {{ .Release.Namespace }} $POD_NAME -o jsonpath="{.spec.containers[0].ports[0].containerPort}")
+  kubectl -n {{ .Release.Namespace }} port-forward $POD_NAME "8080:$CONTAINER_PORT"
   ```
   {{- $listNum = add1 $listNum }}
 {{- end }}
 
-{{ $listNum }}. {{ "Visit the " }}{{ .Chart.Name | camelcase }}{{ " application at: " }}
+{{ $listNum }}. Visit the application at:
   ```
 {{- if .Values.core.route.enabled }}
   echo {{ ternary "https" "http" .Values.core.route.tls.enabled }}://$(oc get route -n {{ .Release.Namespace }} {{ include "cryostat.fullname" . }} -o jsonpath="{.status.ingress[0].host}")
 {{- else if .Values.core.ingress.enabled }}
 {{- range $host := .Values.core.ingress.hosts -}}
   {{- range .paths }}
-  {{ ternary "http" "https" (empty $.Values.core.ingress.tls) }}://{{ $host.host }}{{ .path }}
+  echo {{ ternary "http" "https" (empty $.Values.core.ingress.tls) }}://{{ $host.host }}{{ .path }}
```

Comment:
The `echo` doesn't seem necessary here since all the substitutions are being done by helm - on line 51 the `echo` is needed because there is a shell command `$(oc get route)` emitted as part of the instructions. I guess it seems nice to make these all look similar with an `echo` in front, but I'd rather only put that there if it's actually needed for something.

Reply:
Ah yes, make sense! I guess that applies for other cases where environment variables are present. Thanks!